### PR TITLE
docs: add comment about high 32bit range and signed uids

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: docs/UIDS-GIDS.md [hunk 1]
```diff
@@ -237,7 +237,8 @@ the artifacts the container manager persistently leaves in the system.
 |                 65535 | 16bit `(uid_t) -1`    | Linux         |                               |
 |          65536…524287 | Unused                |               |                               |
 |     524288…1879048191 | Container UID ranges  | `systemd`     | `nss-mymachines`              |
-| 1879048192…4294967294 | Unused                |               |                               |
+| 1879048191…2147483647 | Unused                |               |                               |
+| 2147483648…4294967294 | HIC SVNT LEONES       |               |                               |
```
Comment (floppym) [inline docs/UIDS-GIDS.md:241]: I have never seen the phrase "HIC SVNT LEONES", and had to Google it. I suggest replacing this with something simpler.

Comment (poettering) [reply docs/UIDS-GIDS.md:241]: But why else did I have latin in school if not for the purpose of showing off with it in some technical documentation?

**Comment by mrc0mmand** [comment]
The `TEST-31-OOMPOLICY` fail in CentOS CI is irrelevant and is tracked in https://github.com/systemd/systemd/issues/12425.

**Comment by keszybz** [comment]
illuc ire leones

**Pull request merged**